src/stores/SessionStore.js

```diff
@@ -0,0 +1,63 @@
+import dis from '../dispatcher';
+import EventEmitter from 'events';
+
+/**
+ * A class for storing application state to do with the session. This is a simple flux
+ * store that listens for actions and updates its state accordingly, informing any
+ * listeners (views) of state changes via the 'update' event.
+ */
+function SessionStore() {
```

Comment:
Sure, but I thought we were moving away from those. Unless we're only moving away from them for React components?